app/components/channel_icon.js

```diff
@@ -85,6 +87,13 @@ export default class ChannelIcon extends React.PureComponent {
                     style={[style.icon, unreadIcon, activeIcon, {fontSize: size}]}
                 />
             );
+        } else if (isBot) {
+            icon = (
+                <Icon
+                    name='robot'
+                    style={[style.icon, unreadIcon, activeIcon, {fontSize: size}, style.iconBot]}
```

Comment:
change the icon size with the `size` property instead of the `fontSize` style

Reply:
The icons above and below use `fontSize`. If I do change it to `size` I get an error. Maybe I am misunderstanding?